src/core/datatypes/trees/Tree.cpp

```diff
@@ -2185,6 +2162,29 @@ void Tree::setTaxonObject(const std::string& current_name, const Taxon& new_taxo
 }
 
 
+void Tree::suppressOutdegreeOneNodes(bool reindex, bool replace)
+{
+    // If reindex is false, then suppressOutdegreeOneNodes will either
```

Comment:
I don't get this - what is the point of having a function argument that will crash the code ?

Reply:
Great point. It looks like the only place where that argument can ever be set to `false` is in `Tree::reroot`, and it's not super clear to me why that method calls `suppressOutdegreeOneNodes` anyway. I'm keeping the call in there in case some behavior depends on it, but eliminating the `reindex` argument.